pkg/controllers/rollout/controller_test.go

```diff
@@ -35,6 +38,30 @@ var (
 	cluster3 = "cluster-3"
 	cluster4 = "cluster-4"
 	cluster5 = "cluster-5"
+
+	cmpOptions = []cmp.Option{
+		cmp.AllowUnexported(toBeUpdatedBinding{}),
+		cmpopts.EquateEmpty(),
+		cmpopts.IgnoreFields(fleetv1beta1.ClusterResourceBinding{}, "TypeMeta"),
+		cmpopts.IgnoreFields(metav1.ObjectMeta{}, "ResourceVersion"),
+		cmpopts.SortSlices(func(b1, b2 fleetv1beta1.ClusterResourceBinding) bool {
+			return b1.Name < b2.Name
+		}),
+		cmpopts.IgnoreFields(metav1.Condition{}, "Message"),
+		cmpopts.SortSlices(func(c1, c2 metav1.Condition) bool {
+			return c1.Type < c2.Type
+		}),
+		cmp.Comparer(func(t1, t2 metav1.Time) bool {
```

Comment:
Hi Zhiying! Just being curious, would cmpopts.EquateApproxTime work here?

Reply:
Also, for confirmation, the case where the function will return true if either time is zero is an intentional one, right?

Reply:
yes, the statusCmpOptions will be used in multiple places. Some of them do not need to compare the timestamp though, so i skip checking them by setting as true.

Could split them in a separate PR.